app/migrations/Version20150331151848.php

```diff
@@ -23,30 +23,19 @@ public function up(Schema $schema)
 
         $this->addSql(
             'CREATE TABLE User (
-                id INT AUTO_INCREMENT NOT NULL,
-                username VARCHAR(255) NOT NULL,
-                username_canonical VARCHAR(255) NOT NULL,
-                email VARCHAR(255) NOT NULL,
-                email_canonical VARCHAR(255) NOT NULL,
-                enabled TINYINT(1) NOT NULL,
-                salt VARCHAR(255) NOT NULL,
-                password VARCHAR(255) NOT NULL,
-                last_login DATETIME DEFAULT NULL,
-                locked TINYINT(1) NOT NULL,
-                expired TINYINT(1) NOT NULL,
-                expires_at DATETIME DEFAULT NULL,
-                confirmation_token VARCHAR(255) DEFAULT NULL,
-                password_requested_at DATETIME DEFAULT NULL,
-                roles LONGTEXT NOT NULL COMMENT \'(DC2Type:array)\',
-                credentials_expired TINYINT(1) NOT NULL,
-                credentials_expire_at DATETIME DEFAULT NULL,
-
-                UNIQUE INDEX UNIQ_2DA1797792FC23A8 (username_canonical),
-                UNIQUE INDEX UNIQ_2DA17977A0D96FBF (email_canonical),
-
-                PRIMARY KEY(id)
-            )
-            DEFAULT CHARACTER SET utf8 COLLATE utf8_unicode_ci ENGINE = InnoDB'
+              id INT AUTO_INCREMENT NOT NULL, 
```

Comment:
@tijsverkoyen ik vind het persoonlijk al vreemd dat die migration er nota bene staat. In de README staat er dat je `doctrine:migrations:migrate` moet uitvoeren maar volgens mij is het logischer dat je `doctrine:schema:create` moet doen. Dan hebben we geen SQL te 'maintainen' en hoeft die migration dus niet aangepast te worden. Geen idee waarom dat ooit gekozen is om dat zo te doen